### PR TITLE
docs: LDAP separate credentials file

### PR DESCRIPTION
**Pull request by mbshields**
**What type of PR is this?**

documentation

**Which issue does this PR fix**:

none

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rchincha** [comment]
@laurentiuNiculae pls review this also

File: docs/articles/authn-authz.md [hunk 1]
```diff
@@ -131,22 +132,38 @@ zot supports integration with an LDAP-based authentication service such as Micro
...
-| `userGroupAttribute` | Attribute name used to obtain groups to which a user belongs.                                            |
-| `bindDN`        | Base Distinguished Name for the LDAP search.                                     |
-| `bindPassword`  | Password of the bind LDAP user.                                                  |
+| `userAttribute` | Attribute name used to obtain the username.                                      |
+| `userGroupAttribute` | Attribute name used to obtain groups to which a user belongs.               |
 | `skipVerify`    | Skip TLS verification.                                                           |
 | `subtreeSearch` | Set to `true` to expand the scope for search to include subtrees of the base DN. |
 
+
+A local file contains the bind credentials for the LDAP server, as shown in the following example.
```
Comment (rchincha) [inline docs/articles/authn-authz.md:150]: Credentials for the LDAP server are specified in a separate file, as show ...
This model allows for separation of configuration and credentials.
(rephrase pls)

File: docs/general/whats-new.md [hunk 2]
```diff
@@ -30,6 +30,10 @@
 
 - The validity of an image's signature can be [verified](../articles/verifying-signatures.md) by zot. Users can upload public keys or certificates to zot.
 
+### LDAP credentials stored apart from configuration
```
Comment (rchincha) [inline docs/general/whats-new.md:33]: "apart" -> "separate"

(maybe add this here also)

This model allows for separation of configuration and credentials.
(rephrase pls)

Comment (mbshields) [reply docs/general/whats-new.md:33]: Changed:  "LDAP credentials stored separately from configuration"

**Review by rchincha — APPROVED** [review]
lgtm

**Comment by rchincha** [comment]
What's new -> LDAP change, say that Credentials are now moved to a separate file (it is a breaking change)

**Review by rchincha — APPROVED** [review]
lgtm

**Pull request merged**